README.md

```diff
@@ -109,6 +109,10 @@
 
 </details>
 
+## Links
+
+- rustdoc docs: https://xithrius.github.io/twitch-tui/twt/
```

Comment:
I don't believe we'll need the binary name there.

```suggestion
- Documentation: https://xithrius.github.io/twitch-tui/
```

Reply:
@Xithrius index.html is generated under target/doc/twt so I think we need it, based on my testing at least

Reply:
hmmmmmmmmm

Alright. Let's at least change this to the following:

```md
- Documentation: https://xithrius.github.io/twitch-tui/twt
```